{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -22,9 +22,21 @@ LASTDRAFT = submitted
 environment:
 	mamba env create -f environment.yml
 
-## Update the environment in case of changes to dependencies
-environment-update:
-	mamba env update --name $(CONDA_ENVIRONMENT) --file environment.yml
+## Freeze the environment (fix each installed version) to ensure reproducibility
+## The docker container needs linux, but other recipes can be output as well with:
+## -p osx-64 -p osx-arm64 -p win-64;
+environment-freeze:
+	mamba lock -f environment.yml -p linux-64  \
+	mv conda-lock.yml docker/conda-lock.yml
+
+## Create a static docker image from a working environment (especially helpful for R&R)
+docker-image: 
+	cd docker; \
+	docker build --quiet --tag $(PROJECT_NAME) .
+
+## Run the docker image
+docker-run:
+	docker run -p 8888:8888 --e NB_GID=1001 -v ${PWD}:/home/jovyan/work -it --rm $(PROJECT_NAME) 
```

Comment:
this line is a remnant from copying configs from other docker containers. This container runs from the root user, so there's not necessarily a reason to mount the project directory into the "jovyan" home folder. We should rethink the best place for this

Reply:
done. jovyan is no more